hack/make.ps1

```diff
@@ -363,6 +379,10 @@ Try {
     $root = $(Split-Path $MyInvocation.MyCommand.Definition -Parent | Split-Path -Parent)
     Push-Location $root
 
+    # Create the bundles directory if it doesn't exist
+    $bundlesDir = $root + "\bundles"
+    if (-not (Test-Path $bundlesDir)) { New-Item $bundlesDir -ItemType Directory | Out-Null }
```

Comment:
Can we change to the equivalent of `mkdir -p` here? Then we can also get rid of the intermediate variable

I think that's something like;

```powershell
New-Item -Force "$root\bundles" -ItemType Directory | Out-Null
```


Actually, I see the `Push-Location` above, so we could probably even use a relative path (I see that was was used previously)

```powershell
New-Item -Force ".\bundles" -ItemType Directory | Out-Null
```

Reply:
I did make the change to the latter by making it equivalent to "mkdir -p". However the intermediate variable is then used at other places as well. Making it an explicit value instead of the variable will require us to still set this variable in other places for those usages [or convert those in multiple places to use explicit paths]. But more important, I am afraid it will mask the intent [that it is the same directory which is being referred to in all those disparate locations] and make the code hard to maintain. So I would argue for leaving this as-is.